src/classes/UTIL_OrgTelemetry_SVC.cls

```diff
@@ -48,6 +48,8 @@ public without sharing class UTIL_OrgTelemetry_SVC {
         STG_SettingsService stgSvc = new STG_SettingsService();
         Addr_Verification_Settings__c addrVerifSettings = UTIL_CustomSettingsFacade.getAddressVerificationSettings();
 
-        System.FeatureManagement.setPackageBooleanValue('IsEnabled_CustomizableRollups', (stgSvc.stgCRLP.Customizable_Rollups_Enabled__c == true));
+        if (!Test.isRunningTest()) {
```

Comment:
We'll need a comment.

Reply:
Definitely. This should work fine even in unit tests. Also, there will be additional changes coming to this class that will include a new `UTIL_OrgTelemetry_TEST`class.

Reply:
I added this because I was getting an error when testing my new `@future` function that does CRLP deployments. I removed this check, we'll see if the tests pass.

Reply:
@lparrott  Very interesting. I've been unable to reproduce this in Cumulus. There is the open bug related to this issue, but it hasn't been a blocker for us because I haven't seen it in Cumulus yet. 
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000004rOYEIA2/view

Are you getting this only in your local environment, but not in automated builds?

Reply:
Just the opposite: I'm getting the error ONLY in automated builds (which made it tough to track down!).
From what I can tell, it's because my function doing the metadata deploy is a `@future` method, that's the only difference between my code and what you've built.

Reply:
@lparrott  Would you be willing to try using a Queuable instead of @future to see if that resolves the issue?

Reply:
I got the same error (again, only during the automated build) using Queuable, so I'm going to revert that change.
I'm following the issue you linked to in GUS, hopefully it gets resolved sooner rather than later? For now I'm going to keep the isRunningTest check.